### DISCUSSION TITLE
What are the advantages of using awspostgres connection?

### DISCUSSION THREAD

**Discussion by matthewoflynn** [cyrilgdn/terraform-provider-postgresql#41] [Q&A] [answered]
From the [docs](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs#gocloud), it states that:

> GoCloud simplifies connecting to AWS/GCP hosted databases, managing any proxy or custom authentication details.

If I understand correctly, the `gcppostgres` option helps because using the GCP authentication proxy is mandatory however for RDS I don't understand the advantage of using the `awspostgres` connection.

Why would someone would want to use this versus the default connection type of lib/pq? 

Is it more secure or is it possible to connect using [IAM Authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html) for example?

**Comment by cyrilgdn** [comment, marked as answer]
@matthewoflynn To be honest I don't really know. I read somewhere that it helps to manage AWS certificates but several users complained about certificates troubles when using it.

I'm about to remove it from our documentation but I'll double check that before.

**Reply by matthewoflynn** [reply]
No bother @cyrilgdn, thanks for getting back to me.
